bundle-workflow/src/test_workflow/integ_test/integ_test_suite.py

```diff
@@ -88,7 +88,11 @@ def _execute_integtest_sh(self, endpoint, port, security):
             self.component.name, self.repo.dir
         )
         if os.path.exists(script):
-            cmd = f"{script} -b {endpoint} -p {port} -s {str(security).lower()} -v {self.bundle_manifest.build.version}"
+            cmd = (
+                f"{script} -b {endpoint} -p {port} -s {str(security).lower()} -v {self.bundle_manifest.build.version} -d {self.test_config.working_directory}"
```

Comment:
We can have `cmd` pulled out as it's common around `if` and `else`. Append `-d` based on the check later